utils/data.py

```
@@ -0,0 +1,73 @@
+import json
+import logging
+from pathlib import Path
+from typing import Any, List, Dict, Optional
```

Ruff (0.14.13)

`typing.List` is deprecated, use `list` instead

(UP035)